node_modules/.yarn-integrity

```diff
@@ -1,10 +1,1122 @@
 {
```

Comment:
Should we add this file to the `.gitignore`?

Reply:
Yes, let me update.